src/scripts.js

```diff
@@ -76,21 +76,22 @@ function generateRandomUserId() {
 function displaySleepStatus() {
   sleep.checkUserRestedByDate(user.id, currentDate)
   if (sleep.isRested === true) {
-    $('#sleep-status').attr('src', '../images/ghost-happy.svg');
+    $('#sleep-status').attr({src: '../images/ghost-happy.svg', alt: 'happy ghost icon'});
+    $('#sleep-staus')
```

Comment:
Does adding an alt tag in here reflect in the HTML?

Reply:
Yup, this function is evaluating which image src to load into the DOM. I add alt tags to both versions of the image (happy ghost vs sad ghost) 

Did the same thing for the empty vs full water glass image